### PR TITLE
cmd/updatehub/main.go: server address sanitation

### PR DESCRIPTION
**Pull request by giulianvivan**
Signed-off-by: Giulian Gonçalves Vivan <giulian@ossystems.com.br>

### CODE REVIEW

File: cmd/updatehub/main.go [hunk 1]
```diff
@@ -132,3 +140,23 @@ func main() {
 
 	os.Exit(d.Run())
 }
+
+func sanitizeServerAddress(address string, disableHTTPS bool) (string, error) {
+	a := address
+	if !strings.HasPrefix(a, "http://") && !strings.HasPrefix(a, "https://") {
+		a = "https://" + a
```
Comment (otavio) [inline cmd/updatehub/main.go]: @giulianvivan I believe the prefix should be changed depending on disable-ssl option ... or am I missing something?

File: cmd/updatehub/main.go [hunk 2]
```diff
@@ -132,3 +140,23 @@ func main() {
...
+	}
+
+	serverURL, err := url.Parse(a)
+	if err != nil {
+		return "", err
+	}
+
+	if disableHTTPS {
```
Comment (otavio) [inline cmd/updatehub/main.go]: In fact is there any real need for this option? couldn't this to be inferred from the server URL?

Comment (giulianvivan) [inline cmd/updatehub/main.go]: I've based the implementation on the old agent. I don't remember the reason for it. I'm changing this in a further commit.

File: cmd/updatehub/main.go (hunk 1 above)
Comment (giulianvivan) [inline cmd/updatehub/main.go]: The disableHTTPS option always overwrite the "scheme" passed on the URL, so it will work on both cases.

**Pull request merged**